brave/brave-core
# NFT Details Screen

This PR adds functionality to display NFTs in portfolio and ability to navigate and view NFT details

Resolves https://github.com/brave/brave-browser/issues/22623

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Add an NFT as a custom token by click `Visible Assets` button below portfolio assets. When you complete the process and click Done button in the modal, the NFT will appear in the portfolio assets list
2. Click on the NFT to navigate to NFT details screen

https://user-images.githubusercontent.com/48117473/165968511-53a64e83-e7bd-419f-8870-44612e1c2dc9.mov
